.github/workflows/pr-tests.yml

```diff
@@ -63,6 +65,8 @@ jobs:
         path: |
           ~/pilcom/node_modules
         key: ${{ runner.os }}-pilcom-node-modules
+    - name: Install Rust toolchain 1.72
+      run: rustup toolchain install 1.72-x86_64-unknown-linux-gnu
```

Comment:
Doesnt it get automatically installed because of our rust toolchain file?

Reply:
I think yes, see this error: https://github.com/powdr-labs/powdr/actions/runs/6036953653/job/16380264860

Reply:
For this commit: https://github.com/powdr-labs/powdr/pull/550/commits/46b3ea7e75d745a58ef110035436f3d436b9fa0f